src/main/java/thePackmaster/powers/serpentinepack/TaintedEnemy.java

```diff
@@ -25,18 +25,16 @@ public TaintedEnemy(AbstractCreature owner, int amount) {
     public void updateDescription() {
         if (this.amount == 1) {
             this.description = DESCRIPTIONS[0];
-        } else if (Settings.language == Settings.GameLanguage.ZHS) {
-            this.description = DESCRIPTIONS[1] + this.amount*2 + DESCRIPTIONS[2];
```

Comment:
In the old code, `this.amount*2` was the factor that debuffs were multiplied by. In the new code, it's `this.amount+1`.

Among other things, this means that the ZHS description for 2+ stacks was already using the actual multiplication factor (instead of the number of stacks). With my changes, both languages now use the actual multiplication factor for 2+ stacks, so I got rid of the language-specific if/else and I think the ZHS strings won't even need to change.